src/libs/map-engine.ts

```diff
@@ -83,7 +83,7 @@ export default class MapEngine {
                     },
                     {
                       featureType: 'road',
-                      elementType: 'labels',
+                      xelementType: 'labels',
```

Comment:
just double checking whether this is correct

Reply:
@simoexpo Can you confirm pls? I noticed that there is no `elementType`

Reply:
Back to `elementType`